### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.3 - es. 2 (7 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#273] [Esercizi M-Z]
![es.2](https://github.com/user-attachments/assets/6221c179-3e8a-4220-8921-52f54bc37f13)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/6c4b036b-94aa-49d6-8f2e-a1d7b71592ea)
